cypress/README.md

```diff
@@ -0,0 +1,47 @@
+### cypress in manageiq-ui-classic
+
+#### Run
```

Comment:
There's no mention that server has to be running. Otherwise :+1:

Reply:
Added :)